---
title: 例外 Message4 の完了 |Microsoft Docs
ms.custom: ''
ms.date: 06/08/2017
ms.prod: biztalk-server
ms.reviewer: ''
ms.suite: ''
ms.tgt_pltfrm: ''
ms.topic: article
helpviewer_keywords:
- messages, completing exception flow
- exceptions, completing message
ms.assetid: 689ccfff-d4c3-415f-93df-00785a3882a9
caps.latest.revision: 6
author: MandiOhlinger
ms.author: mandia
manager: anneta
ms.openlocfilehash: 4f30cb22bad77515927b0107cfa6e54aa3a3fbf7
ms.sourcegitcommit: 381e83d43796a345488d54b3f7413e11d56ad7be
ms.translationtype: MT
ms.contentlocale: ja-JP
ms.lasthandoff: 05/07/2019
ms.locfileid: "65356824"
---
# <a name="completing-the-exception-message"></a>例外完了メッセージ
### <a name="to-complete-the-exception-message"></a>例外メッセージを完了するには  
  
-   例外のフローを完了するには追加、**送信**図形をエラー メッセージ _ 3 を監視対象フォルダーにリダイレクトします。  
  
## <a name="see-also"></a>参照  
 [例外のキャッチ ブロックを追加する方法](../core/how-to-add-a-catch-exception-block2.md)   
 [BizTalk Server 例外処理の使用](../core/using-biztalk-server-exception-handling4.md)